health.go

```diff
@@ -30,7 +31,11 @@ type (
 
 // Handler returns an http.Handler
 func Handler(opts ...Option) http.Handler {
-	h := &health{make(map[string]Checker)}
+	h := &health{
```

Comment:
Since the initializations look the same, it might be more legible to add the fields for which the maps are being initialized. 

I'm actually not opinionated on this though since order needs to be guaranteed when fields are not specified. 

Thoughts ?

```
h := &health{
 		checkers:  make(map[string]Checker),
 		observers: make(map[string]Checker),
 	}
```

Reply:
@karthikmuralidharan Yeah, makes sense. I've changed it. Please, take a look.